Makefile

```diff
@@ -157,16 +172,30 @@ check-licenses: licenses-up-to-date ## Check licenses are compliant with our res
 unit-test:
 	go test -race -cover $(GO_UNIT_TEST_FOLDERS)
 
-.PHONY: int-test
-int-test: ENVTEST_ASSETS_DIR = $(shell pwd)/testbin
-int-test: ENVTEST_K8S_VERSION = 1.26.1
-# magical env that if specified makes the test output 0 on successful runs
-# https://github.com/onsi/ginkgo/blob/master/ginkgo/run_command.go#L130
-int-test: export GINKGO_EDITOR_INTEGRATION="true"
-int-test: generate manifests ## Run integration tests. Sample with labels: `make int-test label=AtlasProject` or `make int-test label='AtlasDeployment && !slow'`
+test/int: envtest integration-tests
+	AKO_INT_TEST=1 $(GINKGO)
+
+test/int/clusterwide: envtest integration-tests
+	AKO_INT_TEST=1 $(GINKGO)
+
+integration-tests:
```

Comment:
Why remove the doc comment?

Should we leave `int-test` as an alias to ease the transition?